go/properties.json

```diff
@@ -0,0 +1,19 @@
+{
+    "methodProperties": [
+      {
+        "methodName": "treatedBehaviour",
+        "flagType": "treated",
+        "argumentIndex": 0
+      },
+      {
+        "methodName": "controlBehaviour",
+        "flagType": "control",
+        "argumentIndex": 0
+      },
+      {
+        "methodName": "commonBehaviour",
+        "flagType": "treated",
+        "argumentIndex": 1
+      }
+    ]
+  }
```

Comment:
Ok, I will do that. But like any specific reason for it? Just curious.

Reply:
Ending files with new line is considered best practice. One reason I know of is because command line tools might otherwise merge the last and first line of two files when processing a list of files. Also, some programs will fail to parse the last line entirely if it doesn't end in newline (not that bizarre behavior, actually, since POSIX does define a line as "A sequence of zero or more non- <newline> characters plus a terminating <newline> character.")

Reply:
Doing.